explore-htap.md

```diff
@@ -0,0 +1,104 @@
+
+- TiSpark
+
+    - If your data needs to be analyzed with Spark, deploy TiSpark (Spark 3.x is not currently supported). For specific process, see [TiSpark User Guide](/tispark-overview.md).
+
+<!--    - Real-time stream processing
+  - If you want to build an efficient and easy-to-use real-time data warehouse with TiDB and Flink, you are welcome to participate in Apache Flink x TiDB meetups.-->
+
+## Prerequisites for data
+
+After TiFlash is deployed, data replication does not automatically begin. You need to manually specify the tables to be replicated to TiFlash.
+
+- If there is no data in the TiDB Cluster, migrate the data to TiDB first. For detailed information, see [data migration](/migration-overview.md).
+- If the TiDB cluster already has the replicated data from upstream, after TiFlash is deployed, data replication does not automatically begin. You need to manually specify the tables to be replicated to TiFlash. For detailed information, see [Use TiFlash](/tiflash/use-tiflash.md).
+
+## Data processing
+
+With TiDB, you can simply enter SQL statements for queries or write requirements. For the tables to be replicated to TiFlash, TiDB chooses the best execution freely by the front-end optimizer.
+
+> **Note:**
+> 
+> MPP mode of TiFlash is enabled by default. When an SQL statement is executed, TiDB automatically determines whether to run in MPP mode through the optimizer.
+>
+> - To disable MPP mode of TiFlash, set the value of [tidb_allow_mpp](/system-variables.md#tidb_allow_mpp-new-in-v50) system variable to `OFF`.
+> - To forcibly enable MPP mode of TiFlash for query execution, set the value of [tidb_allow_mpp](/system-variables.md#tidb_allow_mpp-new-in-v50) and [tidb_enforce_mpp](/system-variables.md#tidb_enforce_mpp-new-in-v51) to `ON`.
+> - To see whether to use MPP mode when TiDB execute queries, see [Explain Statements in the MPP Mode](/explain-mpp.md#explain-statements-in-the-mpp-mode). If the output of `EXPLAIN` statement includes `ExchangeSender` and `ExchangeReceiver` operator, MPP mode is activated.
+
+## Performance monitoring
+
+When using TiDB, you can monitor the running status and check TiDB performance by the following methods:
+
+- [TiDB Dashboard](/dashboard/dashboard-intro.md): you can see the overall running status of the TiDB cluster, analyse distribution and trends of read and write traffic, and learn the detailed execution information of slow queries.
+- [Monitoring system (Prometheus & Grafana)](/grafana-overview-dashboard.md): you can see the monitoring parameters of TiDB cluster-related componants including PD, TiDB, TiKV, TiFlash,TiCDC, and Node_exporter.
+
+To see the alert rules of TiDB cluster and TiFlash cluster, see [TiDB cluster alert rules](/alert-rules.md) and [TiFlash alert rules](/tiflash/tiflash-alert-rules.md).
+
+## Troubleshooting
+
+If you have issues while using TiDB, refer to the following documents:
```

Comment:
This seems very generic and not related directly to HTAP

Reply:
those sentences are still necessary contents for introduction of TiFlash and pretty core functionality of HTAP.